test.sh

```diff
@@ -56,11 +56,11 @@ bsource=$(echo $source | base64 -w0)
 DOCKER_FLAGS="-vcg-ast" docker run -e DOCKER_FLAGS --rm $dockerId $bsource | grep -q "__key"
 
 # Check -asm (DMD-only)
-if [[ ! $dockerId =~ "ldc" ]] ; then
-    source='void main() { int a; }'
-    bsource=$(echo $source | base64 -w0)
-    DOCKER_FLAGS="-asm" docker run -e DOCKER_FLAGS --rm $dockerId $bsource | grep -q "_Dmain"
-fi
+# if [[ ! $dockerId =~ "ldc" ]] ; then
+#     source='void main() { int a; }'
+#     bsource=$(echo $source | base64 -w0)
+#     DOCKER_FLAGS="-asm" docker run -e DOCKER_FLAGS --rm $dockerId $bsource | grep -q "_Dmain"
+# fi
```

Comment:
Requires #57

Reply:
You can drop this commit, now that #57 is merged.